profile/nodejs.sh

```diff
@@ -1,11 +1,13 @@
 calculate_concurrency() {
   MEMORY_AVAILABLE=${MEMORY_AVAILABLE-$(detect_memory 512)}
   WEB_MEMORY=${WEB_MEMORY-512}
-  WEB_CONCURRENCY=${WEB_CONCURRENCY-$((MEMORY_AVAILABLE/WEB_MEMORY))}
-  if (( WEB_CONCURRENCY < 1 )); then
-    WEB_CONCURRENCY=1
+  if [ -z "$WEB_CONCURRENCY" ]; then
```

Comment:
I'd like to see a comment here that explains why this is done ("user has no WEB_CONCURRENCY set, so we compute it")